### ISSUE TITLE
updateContent one entry doesn't work :'(

### ISSUE THREAD

**Issue by KillianLeroux** [sleiman/airtable-php#43]
Hi!

I have a problem when I try to update one entry with updateContent(), I follow the doc:

```
$tableName = 'Companies';
$recordId = 'recFjrQs8zRZU80kN';
$lastSyncDate = date('Y-m-d H:i:s');

$values = [
        'name'    => 'Banana',
        'last_sync_date'  => $lastSyncDate,
];

$update = $airtable->updateContent("$tableName/$recordId", $values);
// $update = $airtable->updateContent($tableName . '/{' . $recordId . '}', $values); // Same error message
```

I have this error message:
`"type":"TABLE_NOT_FOUND","message":"Could not find table Companies/recFjrQs8zRZU80kN in application app75638392XXXX"`

The curl URL looks like that:
`https://api.airtable.com/v0/app75638392XXXX/Companies%2FrecFjrQs8zRZU80kN`

The Request\content_type value:
`Companies/recFjrQs8zRZU80kN` 

But when I use bulk update it works:
```
$tableName = 'Companies';
$recordId = 'recFjrQs8zRZU80kN';
$lastSyncDate = date('Y-m-d H:i:s');
$values = [];

$values[] = [
    'id'              => $recordId,
    'fields' => [
        'name'    => 'Banana',
        'last_sync_date'  => $lastSyncDate,
    ]
];

$update = $airtable->updateContent($tableName, $values);
```

Can you help me please? 😥

**Comment by glaliberte** [comment]
Hi  @KillianLeroux ,

Sorry for the late reply...

This has been fixed in in v2.4.5:
https://github.com/sleiman/airtable-php/releases/tag/v2.4.5

Can you try updating to see if it fixes the issue?

Thanks

**Comment by KillianLeroux** [comment]
Hi @glaliberte!

No worries about the response time ;)

I test it and it works! Thanks!!
